CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+The format is based on [Common Changelog](https://common-changelog.org/)
+and this project uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Fixed
+- Return an empty list instead of raising an `IndexError` when searches find no results.
+  ([#839](https://github.com/nsidc/earthaccess/pull/839))
+  ([**@jhkennedy**](https://github.com/jhkennedy))
```

Comment:
```suggestion
### Fixed

- Return an empty list instead of raising an `IndexError` when searches find no results.
  ([#526](https://github.com/nsidc/earthaccess/issues/526))
  ([@jhkennedy](https://github.com/jhkennedy))
```